linux/devdoc/platform_linux_requirements.md

```diff
@@ -0,0 +1,51 @@
+# platform_linux
+
+## Overview
+
+`platform_linux` provides initializes the completion_port system.
+
+## Exposed API
+
+```c
+MOCKABLE_FUNCTION(, int, platform_init);
+MOCKABLE_FUNCTION(, void, platform_deinit);
+MOCKABLE_FUNCTION(, COMPLETION_PORT_HANDLE, platform_get_completion_port);
+```
+
+### platform_init
+
+```c
+MOCKABLE_FUNCTION(, int, platform_init);
+```
+
+`platform_init` initializes the completion port object if it's not initialized
+
+**SRS_PLATFORM_LINUX_11_001: [** If the completion port object is NULL, `platform_init` shall call `completion_port_create`. **]**
+
+**SRS_PLATFORM_LINUX_11_002: [** If `completion_port_create` returns a valid completion port object, `platform_init` shall return zero. **]**
+
+**SRS_PLATFORM_LINUX_11_007: [** If the completion port object is non-NULL, `platform_init` shall return zero. **]**
```

Comment:
>completion port object is non-NULL [](http://example.com/codeflow?start=40&length=34)

how is this different from `valid completion port object`  in 

`SRS_PLATFORM_LINUX_11_002`.

Reply:
Ones for the positive and this is for the negative.